src/main/java/com/example/calculate/ActionController.java

```diff
@@ -0,0 +1,44 @@
+package com.example.calculate;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+// RestController - аннотация для создания контроллера без вызова шаблона
+@RestController
+// RequestMapping - аннотация для групировки вызываемых методов по определенной адресной строке
+@RequestMapping("/calculator")
+public class ActionController {
+    private final HomeService homeService;
+    public ActionController(HomeService homeService) {
+        this.homeService = homeService;
+    }
+    // Вызывает дефолтный метод сервиса welcome() из интерфейса HomeServiceInterface из которого получает строку "Добро пожаловать в калькулятор"
+    @GetMapping(path = "/")
```

Comment:
```suggestion
    @GetMapping
```
можно не писать пустой слэш)